api/common/interfaces.go

```diff
@@ -601,7 +604,10 @@ func getSubscribers(s Serverer, params map[string]interface{}) map[string]interf
 		return respPacking(INVALID_PARAMS, "bucket should be a string")
 	}
 
-	subscribers := chain.DefaultLedger.Store.GetSubscribers(topic, uint32(bucket))
+	subscribers, err := chain.DefaultLedger.Store.GetSubscribers(topic, uint32(bucket))
```

Comment:
should deal with the case : subscribers  is a null map  'map[]'

Reply:
Now client side will get an empty map if subscribers is nil, which is expected IMO